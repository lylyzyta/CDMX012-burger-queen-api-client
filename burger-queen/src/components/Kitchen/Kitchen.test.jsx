/* eslint-disable no-undef */
import React from 'react'
import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import KitchenPage from './Kitchen'

const mockedUsedNavigate = jest.fn()

jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'),
  useNavigate: () => mockedUsedNavigate
}))

describe(('Test component Login'), () => {
  test('render component Login', () => {
    const { asFragment } = render(<KitchenPage />)
    const BtnLogout = screen.getByText('LogOut')
    expect(asFragment()).toMatchSnapshot()
    expect(BtnLogout).toBeDefined()
  })
})
