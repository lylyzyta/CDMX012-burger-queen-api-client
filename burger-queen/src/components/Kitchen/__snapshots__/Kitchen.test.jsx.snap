// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test component Login render component Login 1`] = `
<DocumentFragment>
  <div
    class="containerKitchen"
  >
    <div
      class="headerKitchen"
    >
      <img
        alt="logo-icon"
        class="logoAdmin"
        src="Logo_Image.png"
      />
      <button
        class="btnLogout"
      >
        LogOut
      </button>
    </div>
    <article
      class="boxGeneralKitchen"
    >
      <div
        class="lds-ring"
      >
        <div />
        <div />
        <div />
        <div />
      </div>
    </article>
  </div>
</DocumentFragment>
`;
